app/assets/stylesheets/govuk-component/_related-items.scss

```diff
@@ -0,0 +1,32 @@
+.govuk-related-items {
+  border-top: 10px solid $mainstream-brand;
+  padding-top: 5px;
+
+  h2 {
+    @include core-24;
+    font-weight: 600;
+    color: $text-colour;
+    margin-top: 0.3em;
+    margin-bottom: 0.5em;
+  }
+
+  ul {
+    @include core-16;
+    list-style: none;
+    margin: 0 0 1.25em 0;
+    padding: 0;
+    overflow: hidden;
+
+    li {
+      margin-bottom: 0.75em;
+
+      &.related-items-more {
+        font-weight: bold;
+      }
+
+      a {
+        orphans: 2;
```

Comment:
Should this be in the main styles if it just applies to print CSS?

There's a print mixin you can use to just deliver print styles to the print stylesheet in [this Sass file](https://github.com/alphagov/static/blob/31f79a61ca5535ec805c02ff690616823cc32ad9/app/assets/stylesheets/styleguide/_print.scss#L20).